[BUG] unit test failed due to variable name changed

**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

```bash
# github.com/fluid-cloudnative/fluid/pkg/webhook/scheduler/mutating
vet: pkg/webhook/scheduler/mutating/schedule_pod_handler_test.go:764:14: InjectD
atasetAffinity not declared by package common
make: *** [vet] 错误 123
```

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

--- a/pkg/webhook/scheduler/mutating/schedule_pod_handler_test.go
+++ b/pkg/webhook/scheduler/mutating/schedule_pod_handler_test.go
@@ -760,8 +760,7 @@ func TestAddScheduleInfoToPodWithReferencedDataset(t *testing.T) {
 					Name:      "test",
 					Namespace: "ref",
 					Labels: map[string]string{
-						common.InjectSidecarDone:      common.True,
-						common.InjectDatasetAffinity:  common.True,
+						common.InjectServerfulFuse:    common.True,
 						"fluid.io/dataset.done.sched": "required",
 					},
 				},
@@ -949,8 +948,7 @@ func TestAddScheduleInfoToPodWithReferencedDataset(t *testing.T) {
 					Name:      "test",
 					Namespace: "ref",
 					Labels: map[string]string{
-						common.InjectSidecarDone:      common.True,
-						common.InjectDatasetAffinity:  common.True,
+						common.InjectServerfulFuse:    common.True,
 						"fluid.io/dataset.done.sched": "required",
 					},
 				},

